library/src/main/java/com/schibsted/spain/barista/BaristaRecyclerViewActions.java

```diff
@@ -43,4 +44,9 @@ public static void clickRecyclerViewItemChild(@IdRes int recyclerViewId, int pos
     onView(displayedWithId(recyclerViewId)).perform(
         RecyclerViewActions.actionOnItemAtPosition(position, clickChildWithText(text)));
   }
+
+  public static void checkRecyclerViewItemCount(@IdRes int recyclerViewId, int expectedItemCount) {
```

Comment:
As this method is not an action, but an assertion, can you move it to `BaristaAssertions` class? Right now, `BaristaAssertions` is like the god class that has all the assertions but, if it grows too much, we will split it